gratipay/models/team.py

```diff
@@ -67,6 +67,26 @@ def insert(cls, owner, **fields):
         """, fields)
 
 
+    def update(self, **kw):
+      updateable = frozenset(['name', 'product_or_service', 'homepage',
+                              'onboarding_url', 'todo_url'])
+
+      cols, vals = zip(*kw.items())
+      assert set(cols).issubset(updateable)
+
+      cols = ', '.join(cols)
+      placeholders = ', '.join(['%s']*len(vals))
+
+      return self.db.one("""
+
+        UPDATE teams
+           SET ({0}) = ({1})
+         WHERE id = %s
+     RETURNING teams.*::teams
+
+        """.format(cols, placeholders), vals+(self.id,))
```

Comment:
Let's also write to the `events` table with details about the changes - we need admins to be able to view an 'audit log'. I'm digging up the ticket where this was discussed..